docs/source/guides/feature_descriptions.ipynb

```diff
@@ -127,12 +145,24 @@
     "ft.describe_feature(feature)"
    ]
   },
+  {
+   "cell_type": "raw",
+   "id": "27013806",
+   "metadata": {},
+   "source": [
+    ".. note::\n",
+    "\n",
+    "    When setting a description on a column in a DataFrame as described above, you will have to go through the ``columns`` attribute of the Woodwork DataFrame, ``df.ww.columns[col_name].description``, in order to set the column's description in a way that will be propogated to the feature description. \n",
+    "    \n",
+    "    The other option for selecting the series from the DataFrame with ``df.ww[col_name].ww.description`` will create an entirely new Woodwork Series that is not related to the EntitySet from which feature descriptions are built."
```

Comment:
@rwedge Is this along the lines of how you were thinking we could warn users off of going through the `getitem` in woodwork?

Reply:
@tamargrey yes, although I think this note could focus solely on why not to do `df.ww[col_name]` since we've already shown in the example that `df.ww.columns` works.  We could probably skip re-iterating the correct way to do it

Reply:
Pushed an update that puts the bad syntax first. I did keep a quick reference to the correct syntax after the fact, since I worry that the difference between `es['customers'].ww.columns['join_date'].description` and `es['customers'].ww['join_date'].ww.description` may not be immediately obvious.